src/math/Quaternion.js

```diff
@@ -354,15 +353,12 @@ Object.assign( Quaternion.prototype, {
 
 		// assumes direction vectors vFrom and vTo are normalized
 
-		var v1 = new Vector3();
```

Comment:
If you remove `v1` here, the closure is not necessary anymore. It has no benefit for primitive values.

Reply:
Good point. Let's see if removing the dependencies are approved. If so, I will make your requested change. Thanks for reviewing.

Reply:
#16039